app/presenters/edition_presenter.rb

```diff
@@ -89,9 +88,9 @@ def access_limited
       else
         {
           access_limited: {
-            users: access_limit.users,
-            organisations: access_limit.organisations,
-            auth_bypass_ids: access_limit.auth_bypass_ids,
+            users: access_limit&.users,
+            organisations: access_limit&.organisations,
```

Comment:
I'm not clear why you've added the ampersands here?

Reply:
I've added ampersands here to signify that access_limit could be nil. Adding an `&` after access_limit means that if access_limit is `nil` then calling users or organisations on this value wont throw a method error and instead return nil.

This is needed as the check here https://github.com/alphagov/publishing-api/pull/1625/files#diff-5e3b9d2d7f141f5b5d7383dac0157593R87 means that there doesn't necessarily need to be an access_limit if an edition has `auth_bypass_ids`, the code here will be altered and removed as we only need it to keep this way to preserve the legacy way of sending bypass_ids to content-store.

Reply:
Ahhhh thanks for the explanation. I'd forgotten about that.